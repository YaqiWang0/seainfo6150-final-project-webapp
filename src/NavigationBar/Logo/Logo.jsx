import React from "react"
import { useHistory, useLocation } from "react-router-dom";
import style from "./Logo.module.css"

const Logo = (props) => {
    let history = useHistory();

    const click = () => {
        history.push(
            "/",
            {
                login: props.login,
                userId: props.credential.userId,
                email : props.credential.email
            }
        )
    }

    return (
        <div className={style.container} onClick={click}>
        <div className={style.pic}>
        </div>
        </div>
    )
}

export default Logo