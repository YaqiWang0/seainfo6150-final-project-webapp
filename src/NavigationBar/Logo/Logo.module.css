.pic {
    background-image: url("../../pictures/Logo.png");
    min-width: 10vmin;
    min-height: 10vmin;
    cursor: pointer;
    background-size: cover;

}

.container {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    padding: 0.5vw;
}

.name {
    color: white;
    font-size: 1vmin;
    font-weight: bold;
}