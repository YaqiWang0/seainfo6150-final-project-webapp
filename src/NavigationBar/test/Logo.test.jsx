
import React from 'react'
import Logo from "../Logo/Logo";


describe("Logo test", () => {

    it('render successfully', function () {
        const {container} = render(
            <Logo />
        );
        expect(container).toMatchSnapshot();
    })
})