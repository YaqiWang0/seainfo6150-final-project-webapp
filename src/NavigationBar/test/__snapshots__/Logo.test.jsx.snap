// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo test render successfully 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="pic"
    />
  </div>
</div>
`;
